bundles/org.openhab.binding.sonyprojector/src/main/resources/OH-INF/i18n/sonyprojector.properties

```diff
@@ -21,16 +21,148 @@ thing-type.config.sonyprojector.ethernetconnection.host.description = Host name
 thing-type.config.sonyprojector.ethernetconnection.model.label = Model
 thing-type.config.sonyprojector.ethernetconnection.model.description = Projector model to be controlled. Default is AUTO.
 thing-type.config.sonyprojector.ethernetconnection.model.option.AUTO = Automatic
+thing-type.config.sonyprojector.ethernetconnection.model.option.VPL-HW60 = VPL-HW60
```

Comment:
For the sonyprojector binding,  it was deliberate to not include options containing model names.

Reply:
Removed.